metashare/api/jobs.py

```diff
@@ -100,5 +92,5 @@ def create_branches_on_github_then_create_scratch_org_job(
         scratch_org=kwargs["scratch_org"],
         user=kwargs["user"],
         repo_url=kwargs["repo_url"],
-        commit_ish=kwargs["commit_ish"],
+        commit_ish=kwargs["task"].branch_name,
```

Comment:
I tried to actually deconstruct all the arguments, to make it possible to test without constructing weird objects with values on them (which is harder in Python than in JS), hence the explicit `commit_ish` kwarg, but I've unbundled the kwargs at this point, so.